scripts/micro_benchmarks.py

```diff
@@ -222,7 +231,7 @@ def __process_arguments(args: list) -> Tuple[list, bool]:
 
 
 def __get_benchmarkdotnet_arguments(framework: str, args: tuple) -> list:
-    run_args = ['--']
```

Comment:
Is this seperator not needed with benchmarks_ci.py runs?

Reply:
Actually, I missed this! This would still be needed for the non-isolated run, where we use `dotnet run`. I'll push a fix.

Reply:
I was wondering why it didn't fail on the previous run. And IIUC, any arguments that `dotnet run` doesn't know how to parse, get passed to the app. But adding `--` ensures that anything after that is always passed to the app.